scripts/bd_to_prod.sh

```diff
@@ -7,6 +7,7 @@ TAG=$(semantics --output-tag)
 if [[ ! -z "$TAG" ]] ; then
   git log --oneline > changelog.md
   gh release create $TAG -F changelog.md
-else 
+  echo \{\"type\":\"Release\",\"repo\":\"${CIRCLE_PROJECT_REPONAME}\",\"buildUrl\":\"${CIRCLE_BUILD_URL}\"\} | curl -X POST "https://listener.logz.io:8071?token=${LOGZ_TOKEN}" -v --data-binary @-
```

Comment:
@saartochner just make sure to add `LOGZ_TOKEN` as an env variable to circleci

Reply:
I think that I already added it but it's better to verify it.

Reply:
Of course I added 😂